src/Provider/Blizzard.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/*!
+* Hybridauth
+* https://hybridauth.github.io | https://github.com/hybridauth/hybridauth
+*  (c) 2017 Hybridauth authors | https://hybridauth.github.io/license.html
+*/
+
+namespace Hybridauth\Provider;
+
+use Hybridauth\Adapter\OAuth2;
+use Hybridauth\Exception\UnexpectedApiResponseException;
+use Hybridauth\Data;
+use Hybridauth\User;
+
+/**
+ * Blizzard Battle.net OAuth2 provider adapter.
+ */
+class Blizzard extends OAuth2
+{
+    /**
+    * {@inheritdoc}
+    */
+    public $scope = '';
+
+    /**
+    * {@inheritdoc}
+    */
+    protected $apiBaseUrl = 'https://us.api.battle.net/';
```

Comment:
@cookieman768 Correct me if i'm wrong, but didn't you hardcoded region into your Provider? What about other regions?
Here is the list of available regions:
* EU
* US
* TW
* KR
* SEA

Reply:
Ahh. I didn’t think about that. Would it be best to have a provider for each region or an option in the config? I don’t think that the other providers have this issue. Let me know what approach I should take. It would be pretty easy to change this for each region.

Reply:
I just looked at the providers, it looks like for WeChat there is a different provider specifically for China. I’ll take that approach and make multiple providers for each region.

Reply:
@cookieman768, yeah `WeChat` approach looks good for me, you can do the same. So this one will be default (with US). I think we need to add EU as well. Others only if there will be request.

Reply:
@ApacheEx , there is only 3 regions, so I think it shouldn't be very difficult to add one extra to completely cover all available regions. You can count this as request for `APAC` region.

Reply:
@Edarlingen - fine for me 😏

Reply:
I'll work on this when I get the chance, for the time being if someone really needs the other regions they can modify this version. I'll get some time in the coming weeks when I finish my finals.